website/docs/reference/change-requests.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Change requests
+---
+
+:::info
+
+The change requests feature was introduced in **Unleash 4.18.0**, and is currently an enterprise only feature in beta
+with a select few customers. If you're an unleash enterprise customer, please contact us if you'd like to try out this 
+feature. 
+
+:::
+
+Feature flagging is a powerful tool, and because it's powerful you sometimes need to practice caution. The ability to have complete control over your production environment comes at the cost of being able to make mistakes in production. Change requests are here to alleviate that fear. Introduced in 4.18.0, you can now group changes in a change request instead of applying them directly to production. This allows you to make changes to multiple feature toggles configuration and status (on/off) grouped together, and applied together at the same time. 
```

Comment:
```suggestion
Feature flagging is a powerful tool, and because it's so powerful, you sometimes need to practice caution. The ability to have complete control over your production environment comes at the cost of the potential to make mistakes in production. Change requests were introduced in version 4.18.0 to alleviate this fear. Change requests allow you to group changes together and apply them to production at the same time, instead of applying changes directly to production. This allows you to make multiple changes to feature toggles and their configuration and status (on/off) all at once, reducing the risk of errors in production.
```